clients/packages/polarkit/src/components/badge/Badge.tsx

```diff
@@ -211,6 +220,56 @@ export const Badge = ({
               </div>
             )}
 
+          {showAvatars.length > 0 && (
+            <div
+              style={{
+                display: 'flex',
+                marginRight: '8px',
+                overflow: 'hidden',
+                height: '22px',
+                flexWrap: 'wrap',
+              }}
+            >
+              {showAvatars.map((url, idx) => (
+                <img
+                  key={idx}
+                  src={url}
+                  style={{
+                    height: 22,
+                    width: 22,
+                    borderRadius: 22,
+                    border: darkmode ? '1px solid #374e96' : '1px solid white',
+                    marginLeft: idx > 0 ? '-6px' : '',
+                    flexShrink: '0',
+                  }}
+                />
+              ))}
+
+              {extraAvatarsCount > 0 && (
+                <div
+                  style={{
+                    backgroundColor: darkmode ? '#2e4070' : '#C9DBF4',
+                    color: darkmode ? '#a6c7ea' : '#4667CA',
+                    height: 22,
+                    width: 22,
+                    borderRadius: 22,
+                    marginLeft: '-6px',
+                    textAlign: 'center',
+                    fontSize: '8px',
+                    lineHeight: '20px',
+                    border: darkmode ? '1px solid #374e96' : '1px solid white',
```

Comment:
Let's try to change the dark mode border to "#2e4070" same as the background. So it's more of a carve-out vs. border as is the case in the light mode.

Reply:
Nice idea, I guess you mean #3E3F43 in that case :)